_data/sites.json

```diff
@@ -2486,6 +2487,18 @@
         ]
     },
 
+    {
+      "name": "CDEK Forward",
+      "url": "https://global.cdek.ru",
+      "difficulty": "hard",
+      "notes": "You need to contact support through email",
```

Comment:
I suggest that we should use this block of translations instead of just the English and Russian version.

```suggestion
      "notes": "Account deletion requires contacting Customer Support.",
        "notes_tr": "Hesabınızın silinmesi için müşteri hizmetleri ile iletişime geçmeniz gerekir.",
        "notes_fr": "Pour supprimer votre compte contactez le support.",
        "notes_it": "La cancellazione account richiede di contattare il servizio clienti.",
        "notes_de": "Account-Löschung erfordert es den Kundensupport zu kontaktieren",
        "notes_pt_br": "Para deletar a sua conta contate a assistência ao cliente",
        "notes_cat": "Per esborrar el seu compte contacti amb l'assistència al client",
        "notes_es": "Para borrar tu cuenta contacta con asistencia al cliente",
```

Reply:
We typically ask people to just translate languages they are fluent on, even if machine translation seems good enough. Are you fluent in all of t hose?

Reply:
> We typically ask people to just translate languages they are fluent on, even if machine translation seems good enough. Are you fluent in all of t hose?

I didn't get this from Google Translate or DeepL. These are from the official sites.json file of this project.

Reply:
Ah, well then. Can you fix the suggestion so it has proper indentation then, please?

Reply:
> Ah, well then. Can you fix the suggestion so it has proper indentation then, please?

Yep,here you go.